### ISSUE TITLE
Unstitched output

### ISSUE THREAD

**Issue by binghama** [cabooster/DeepCAD#13]
I've been running DeepCAD with the following command:

`main.py --GPU 0 --img_h 64 --img_w 64 --img_s 320 --train_epochs 30 --datasets_folder Channel0_Zavg_allT --normalize_factor 1 --lr 0.00005 --train_datasets_size 100`

My data folder contains a single tif file of dimension 1000x512x512 (time, y, x). DeepCAD seems to run as expected and outputs a noise1, noise2, and output file for each small patch in the results folder. At the end, should DeepCAD stitch these tiles into an output with the same dimensions as the input? 

Unfortunately my input file is too large for upload. Let me know if there's an alternate route for sending if needed. Thanks!

**Comment by cabooster** [comment]
Hi binghama. We have released a new version of DeepCAD at [https://github.com/cabooster/DeepCAD-RT](https://github.com/cabooster/DeepCAD-RT) with comprehensive optimization and acceleration. You can try it and find that everything is OK. The current DeepCAD is not under maintenance.

**Comment by binghama** [comment]
Thanks for letting me know @cabooster.
